partner_industry_secondary/migrations/11.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_res_partner_industry(cr):
+    cr.execute(
+        """
+        INSERT INTO res_partner_industry (name, active, create_uid,
+            create_date, write_uid, write_date, parent_id, %s)
+        SELECT name, TRUE as active, create_uid, create_date, write_uid,
+            write_date, parent_id, id as %s
+        FROM res_partner_sector
+        """ % (
+            openupgrade.get_legacy_name('old_id'),
+            openupgrade.get_legacy_name('old_id'),
+        )
+    )
+    cr.execute(
+        """
+        UPDATE res_partner_industry rpi2
+        SET parent_id = rpi.id
+        FROM res_partner_industry rpi
+        WHERE rpi2.parent_id = rpi.%s
+        """ % openupgrade.get_legacy_name('old_id')
+    )
+
+
+def update_res_partner_industry(cr):
+    cr.execute(
+        """
+        UPDATE res_partner rp
+        SET industry_id = rpi.id
+        FROM res_partner_industry rpi
+        WHERE rp.%s IS NOT NULL AND rp.%s = rpi.%s
```

Comment:
No need of the `IS NOT NULL` part

Reply:
It's needed, because in "`AND rp.%s = rpi.%s`" you don't want `NULL = NULL` that returns `TRUE`